Assets/Scripts/Gameplay/UI/IPUIMediator.cs

```diff
@@ -171,13 +172,30 @@ public void CancelConnectingWindow()
         }
 
         /// <summary>
-        /// Sanitize user port InputField box allowing only alphanumerics and '.'
+        /// Sanitize user IP address InputField box allowing only numbers and '.'
         /// </summary>
         /// <param name="dirtyString"> string to sanitize. </param>
         /// <returns> Sanitized text string. </returns>
-        public static string Sanitize(string dirtyString)
+        public static string SanitizeIP(string dirtyString)
         {
-            return Regex.Replace(dirtyString, "[^A-Za-z0-9.]", "");
+            return Regex.Replace(dirtyString, "[^0-9.]", "");
```

Comment:
we don't support "localhost"?

Reply:
No, it gives an `Invalid network endpoint` error